src/chart/AccessibilityManager.ts

```diff
@@ -0,0 +1,94 @@
+import type { LayoutType, TickItem } from '../util/types';
+
+interface ContainerOffset {
+  top: number;
+  left: number;
+}
+
+interface InitiableOptions {
+  coordinateList: TickItem[];
+  mouseHandlerCallback: (e: any) => void;
+  container: HTMLElement;
+  layout: LayoutType;
+  offset: ContainerOffset;
+}
+
+export class AccessibilityManager {
+  private activeIndex = 0;
+
+  private coordinateList: InitiableOptions['coordinateList'] = [];
```

Comment:
personal preference would just be to define these types as their true types and without `InitiableOptions` (can still exist for the sake of the `init` function but these are easier to read if their explicit types are here)